barbeque/storage.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
+# from django.contrib.staticfiles.storage import ManifestFilesMixin
+
+from collections import OrderedDict
+
+from django.conf import settings
+from django.contrib.staticfiles.utils import matches_patterns
+from django.core.files.base import ContentFile
+from django.utils.encoding import force_bytes, force_text
+from django.utils.six import iteritems
+
+
+class StaticFilesStorage(ManifestStaticFilesStorage):
```

Comment:
I'd give this a more explicit name. What about CompactManifestStaticFileStorage?